src/Components/Navbar.js

```diff
@@ -8,13 +8,27 @@ import { Menubar } from 'primereact/menubar'
 function Navbar({ items, start, end }) {
   if (!end) {
     end = (
-      <a href="https://github.com/EddieHubCommunity/LinkFree" aria-label="View source on GitHub">
-        <i className="pi pi-github" style={{ fontSize: '2em' }}></i>
-      </a>
+      <div className="p-d-flex p-jc-center p-ai-center p-pr-2">
+        <div className="p-p-1"> Contribute on </div>
+        <a
+          href="https://github.com/EddieHubCommunity/LinkFree"
+          className="p-p-1"
+        >
+          <i className="pi pi-github" style={{ fontSize: '1em' }}></i>
+        </a>
+        <div>v0.0.0</div>
```

Comment:
Why is this hard-coded though?

Reply:
I will take the code from the footer component.

Reply:
Instead of repeating the code from the footer in the navbar, maybe it would be better to lift the state up to the parent component of both footer and navbar so it only exists as state one place in the app and pass it down as props.

Reply:
> Instead of repeating the code from the footer in the navbar, maybe it would be better to lift the state up to the parent component of both footer and navbar so it only exists as state one place in the app and pass it down as props.

Sure, will work on that :)

Reply:
![image](https://user-images.githubusercontent.com/84982038/142001377-8f4f8f96-80fa-468a-aa11-cf142149e3ac.png)
I am having this error while passing {version} from User.js to Navbar.js as props.

Reply:
@emmalearnscode @krishguptadev, we don't need a footer anymore, right? Cause using the footer, will repeat the contents.

Reply:
For passing that, you would have to add it to the `propTypes` for `Navbar`
```diff
Navbar.propTypes = {
  items: PropTypes.array,
  start: PropTypes.object,
  end: PropTypes.object,
+ version: PropTypes.string.isRequired,
}
```

Reply:
Yes, maybe best to remove the footer.

Reply:
@Vyvy-vi @emmalearnscode Noted!

Reply:
![image](https://user-images.githubusercontent.com/84982038/142428745-03048668-c024-4578-b303-cfe7c0e6ebcf.png)
![image](https://user-images.githubusercontent.com/84982038/142428797-44ca5222-e704-4d1c-b91b-1f1defd8416b.png)
![image](https://user-images.githubusercontent.com/84982038/142428874-9c9b927f-7c66-4ed1-9cb3-4b328a175b4a.png)

While passing the version as props to the Navbar component, I am facing these issues.
Since we are using the "version" in Navbar only, I suggest importing "version" in that component only if it doesn't cause problems.

Reply:
Yah, I think it would make sense to import it only there

Reply:
I removed the footer and added the version in Navbar.
Please check.
![image](https://user-images.githubusercontent.com/84982038/142589945-d47e9c9e-84a9-44ad-98f0-d5c6de50f5e3.png)